charts/ccsm-helm/charts/identity/templates/deployment.yaml

```diff
@@ -20,6 +20,10 @@ spec:
         image: "{{ .Values.image.repository }}:{{ .Values.global.image.tag }}"
         {{- end }}
         env:
+          - name: KEYCLOAK_INIT_OPERATE_SECRET
+            value: "the-cake-is-alive"
+          - name: KEYCLOAK_INIT_OPERATE_ROOT_URL
+            value: "http://localhost:8080"
```

Comment:
@dlavrenuek without this I can't log in is this correct?

Reply:
Without this part users will not be able to login in Operate. If Operate is not configured to use Identity yet, these values can be removed for now.

Reply:
Do you know why then log in didnt work without this?

Reply:
What login do you mean? Login into Operate or Identity? Locally or in the tests?

Reply:
The IT test failed with demo:demo without this setting. But i can also try it again when Im back

Reply:
So not 100% true, what fails is the login with the jwt token.